### ISSUE TITLE
Implementing on Window rtx3080

### ISSUE THREAD

**Issue by Ehteshamciitwah** [ying09/TextFuseNet.pytorch#80]
Hello, Thank you for your wonderful work. I just want to implement your framework on my dataset. I need guidance your in the following points
1. installing your framework on Windows with rtx3080 is showing an error. can you please suggest the installation step for windows
2.  How I can annotate my data for training purposes. 
I am looking for your kind response. 
Thank you

**Comment by Real-YeJ** [comment]
@Ehteshamciitwah 
1. We haven't tried it on Windows yet. I suggest you build our TextFuseNet on Linux. Maybe you can check the detectron2 codebase (https://github.com/facebookresearch/detectron2) to find  an answer.
2. Pls use our pre-trained model to detect your images, and then generate pseudo labels from detection results.
